Client/Clique/Common/Functions/Functions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Functions.swift
+//  Clique
+//
+//  Created by Infinum on 09.12.2022..
+//
+
+import UIKit
+
+enum Functions {
+    
+    enum Alerts {
+        static func alert(fwdMessage: String,viewController: UIViewController){
```

Comment:
Please rename `fwdMessage` to something else. Honestly, I'm not sure what `fwd` stands for 😄